File: CSharp/1285.cs
using System;
using System.Linq;

class Program
{
    static void Main(string[] args)
    {
        int[] n;
        int aux;
        int cont=0;
        string x;

        while(true)
        {
            x=Console.ReadLine();
            if(string.IsNullOrEmpty(x))
            {break;}
            n = x.Split().Select(int.Parse).ToArray();
            aux = n[0];
            while(aux<=n[1])
            {
                if(aux.ToString().ToCharArray().Length==aux.ToString().ToCharArray().Distinct().Count())
                {
                    cont++;
                }
                aux++;
            }
            Console.WriteLine(cont);
            cont=0;
        }

    }
}

File: Python/1285.py
while True:
    try:
        x = input().strip()
        if not x:
            break

        n = list(map(int, x.split()))
        cont = 0
        aux = n[0]

        while aux <= n[1]:
            num_str = str(aux)
            if len(num_str) == len(set(num_str)):
                cont += 1
            aux += 1

        print(cont)
    except EOFError:
        break
